org.kde.ghostwriter.yml

```diff
@@ -0,0 +1,66 @@
+app-id: org.kde.ghostwriter
+runtime: org.kde.Platform
+runtime-version: 5.15-22.08
+base: io.qt.qtwebengine.BaseApp
+base-version: 5.15-22.08
+sdk: org.kde.Sdk
+command: ghostwriter
+rename-icon: ghostwriter
+finish-args:
+- "--socket=wayland"
+- "--socket=x11"
+- "--socket=pulseaudio"
+- "--share=ipc"
+- "--share=network"
+- "--filesystem=home"
```

Comment:
Any reason this is needed? Is the file portal not good enough for this app? If so, you should add a README with a small note explaining the reason here.